Automattic/jetpack
# Shortcodes: do not overwrite Core's Instagram oEmbed in new editor

Fixes #9331

#### Changes proposed in this Pull Request:


When using the block editor, let's rely on core's Instagram embed method. We don't really need to overwrite it to use our own there.

#### Testing instructions:

* Go to the old Jetpack > Modules list and make sure that the Shortcode embeds module is active.
* Create multiple posts:
    - using the new block editor, and pasting the following URL on its own line: `https://www.instagram.com/p/BnMO9vRleEx/`
    - using the new block editor, and pasting the following URL on its own line inside a classic block: `https://www.instagram.com/p/BnMO9vRleEx/`
    - using the new block editor, and pasting the following on its own line in a classic block: `[instagram url=https://www.instagram.com/p/BnMO9vRleEx/]`
* Make sure the embeds work well (no errors on the page or in the browser console) in the editor and on the frontend.

#### Proposed changelog entry for your changes:

* Instagram: make sure Instagram embeds work well in the new block editor.
